src/main/webapp/i18n/en/exam.json

```diff
@@ -345,7 +345,8 @@
                     "totalpointsbiggerequal": "Total points in exam smaller or equal to all mandatory + optional points",
                     "hasExercises": "has",
                     "zeroexercises": "0",
-                    "exercise": "Exercise",
+                    "exerciseStart": "Exercise-group",
+                    "exercise": "exercise",
```

Comment:
don't know how important that is but full stop at the end is missing so that it is consistent with "exercises."

Reply:
Thanks, its refactored now